deploy/helm/sumologic/values.yaml

```diff
@@ -218,6 +218,68 @@ sumologic:
       ## Set provider service (either fluentd or otelcol).
       provider: fluentd
 
+    collector:
+      enabled: false
+
+    multiline:
+      enabled: true
+      start_regex: "^\\[?\\d{4}-\\d{1,2}-\\d{1,2}.\\d{2}:\\d{2}:\\d{2}.*"
+
+    container:
+      enabled: true
+
+    systemd:
```

Comment:
Correct me if I'm wrong, but it looks like there's now no simple way to set e.g. a custom source category on logs from systemd?

It's just an observation - I don't think it's a blocker for this PR, as this can be added in a later PR, right?

Reply:
That's something that the metadata layer does, so we'd have to add it there, ideally while also moving configuration to a template like I've done here. This PR is only concerned with configuration for the log collector.